README.md

```diff
@@ -4,18 +4,58 @@ SourcePauseTool
 A plugin for all your pausing needs.
 
 ## Usage
-Place the compiled `spt.dll` into your mod directory (for example: `hl2`, `portal`, `ep2`), launch the game and type `plugin_load spt` into the developer console. For older engines you will need to place `spt.dll` into the topmost `bin` directory (the one with a lot of different DLL files including `engine.dll`, *not* the one with `client.dll` and `server.dll`).
+*Note: Steam, by default, installs games to `C:\Program Files (x86)\Steam\steamapps\common` on 64-bit Windows, and `C:\Program Files\Steam\steamapps\common` on 32-bit Windows.*
+
+1. Place the corresponding `spt*.dll` in the mod directory, *for example*:
+  * `spt-2013.dll` - Source SDK 2013/SteamPipe.
+  * `spt-bms.dll` - Black Mesa
+  * `spt-oe.dll` - Source SDK 2006/Old Engine.
+  * `spt-p2.dll` - Portal 2
+  * `spt.dll` - Source SDK 2007/Source Unpack/New Engine.
+* Goes in:
+  * `hl2` - Half-Life 2
+  * `episodic` - Half-Life 2: Episode 1
+    * For Old Engine you will need to place `spt-oe.dll` into the topmost `bin` directory; The one with a lot of different DLL files including `engine.dll`, **not** the one with `client.dll` and `server.dll`.
+  * `ep2` - Half-Life 2: Episode 2
+  * `portal` - Portal
+  * `portal2` - Portal 2
+  * `bms` - Black Mesa
+
+2. Launch the game.
+3. Go to `Settings > Keyboard > Advanced` and check `Enable Developer Console`.
+4. Press the tilde key (~) and enter `plugin_load spt` into the developer console. 
+    * To automatically load SourcePauseTool add `plugin_load spt` to the `cfg/autoexec.cfg`.
 
 ## Building
-You will need Visual Studio 2017 or above.
-1. Get the Source SDK code that you wish to build SPT for.
-1. Clone this repository into a folder under `src\utils`, where `src` is the folder with the Source SDK source code. The files from this repository should end up inside `src\utils\SourcePauseTool`.
-1. Inside the `SourcePauseTool` folder do `git submodule update --init --recursive`.
-1. Open `spt.sln` in Visual Studio and build the correct build configuration:
-   - Release OE for Source SDK 2006
-   - Release for Source SDK 2007
-   - Release 2013 for Source SDK 2013
-   - Release P2 for the Portal 2 SDK from alliedmodders.
+You will need Visual Studio 2017 or later, and [Git](https://git-scm.com).
+
+1. Enter the following into cmd:
+```
+git clone -b <SDK> https://github.com/alliedmodders/hl2sdk.git
+# Replace <SDK> with the wanted SDK:
```

Comment:
These comments should be outside of the code block. Right now it looks as if you want people to literally enter them into the command line.